systemd/user/kano-sync.service

```diff
@@ -0,0 +1,23 @@
+#
+# kano-sync.service
+#
+# Copyright (C) 2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPLv2
+#
+# This systemd service will run kano-sync as the current
+# interactive logged in user.
+#
+# DHCP hooks are responsible for triggering
+# the internet-available-user.target
+#
+
+[Unit]
+Description=Kano Sync
+Before=internet-available-user.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/kano-sync --upload-tracking-data --silent
```

Comment:
Also, should we consider sync-ing everything in this service? User data, tracking data, etc.?

Reply:
I would propose to fix one thing at a time. Right now this command fails due to - presumably - some mismatch in the way the new KW API expects this data, but I can't tell for sure.